### DISCUSSION TITLE
WOCHE 01 - Projektstart

### DISCUSSION THREAD

**Discussion by DossohMael** [mausio/corgi-shop-doc#3] [Blogs]
### 1. Projektvision & Teammitglieder
- „Corgi Shop“ soll eine Mobile Applikation werden, die es Kunden ermöglichen soll Hunde der Rasse Corgi sowie Accessoires mit Corgi Motiven zu erwerben. 
- Das Projekt „Corgi Shop“ wird von Niklas Krauth (Backend / Scrum Master), Robin Schwenzfeier (Frontend), Mael Dossoh (PO, CI/CD & Backend) durchgeführt.

### 2. Verwendete Tech-Stacks
- Die App verfügt über ein Front- sowie ein Backend. Diese sollen jeweils mit Hilfe der Frameworks Swift (Frontend) sowie Springboot (Backend) geschrieben werden. 
  Weitere Tech-Stacks die benutzt werden sind Postman, Docker, GitHub.

### 3. Scrum-Setup
- Die Dailies finden Dienstags nach den Vorlesungen statt. 
Die Termine für Retrospektive und das weitere Planning werden noch festgesetzt.
Für das Scrum des Projekts wird Jira benutzt. 
- Das Projekt wurde in 5 Sprints eingeteilt die jeweils ein Semester lang sind.
Im ersten Semester sollen die Corgis und die Accessoires aufgesetzt werden, im zweiten dann die Adminseite, das Check-Out System und die User Seite implementiert werden.

### 4. Sprintbacklog
- Anbei finden Sie noch Screenshots des Kabanboards des Sprintbacklogs. 
![Screenshot 2023-10-23 202523](https://github.com/mausio/corgi-shop-doc/assets/122524882/2e1c3d4c-2f5a-486c-97fb-736cf0be661a)

Danke für Ihre Zeit!

**Comment by Puggingtons** [comment]
Dass Corgies involviert sein werden, war ja klar 😂
Findet ihr es wirklich eine gute Idee, nur 2 Sprints im Verlauf des Projekts zu haben? Ich denke, dass dadurch der Sinn eines "Sprintes" verloren geht.

**Reply by mausio** [reply]
Guten Tag Herr Bader,
Wir haben im Laufe des Projekts 5 Sprints für 2 Semester.
Ist dies Ihrer Meinung nach zu wenig?

**Reply by DossohMael** [reply]
Danke für den Hinweis Herr Bader, es ist uns wohl ein kleiner Zahlendreher unterlaufen.
Es sind, wie Herr Schwenzfeier schon akkurat beschrieben hat, doch 5 Sprints.

Einen wundervollen Tag Ihnen!

**Reply by mausio** [reply]
Frech

**Reply by mausio** [reply]
> Dass Corgies involviert [...]

Corgis*

🫶🏼

**Comment by Phylyp1** [comment]
Corgis waren klar :) Wird die App die gleichen Features haben wie euer Online-Shop ?

**Reply by mausio** [reply]
Grüß Gott,
Es wird ein extra Back End entwickelt mit einer API, Authentifizierung und Checkout.
Der Corgi ist ein Markenzeichen der RMN™ und gehört zu jedem Projekt dazu.
